package.json

```diff
@@ -72,7 +73,7 @@
     "@types/pluralize": "^0.0.29",
     "@types/range-parser": "^1.2.3",
     "@types/rbush": "^3.0.0",
-    "@types/react": "^16.9.34",
+    "@types/react": "^17.0.0",
```

Comment:
We're still using React v16, will using the v17 types cause any problems?

Reply:
back to 16 now